ssl_test.go

```diff
@@ -182,6 +209,8 @@ func assertConnectionTntFail(t testing.TB, serverOpts, clientOpts SslOpts) {
 func assertConnectionTntOk(t testing.TB, serverOpts, clientOpts SslOpts) {
 	t.Helper()
 
+	skipEncrypted(t, serverOpts)
```

Comment:
It would be nice to skip tests in `TestSslOpts` tests before `assert*` start. It is not obvious that `assert*` could skip a test case from the first glance.

Reply:
But you cannot stop a subtest outside the subtest, isn't it? You can only not create a subtest at all. Do you prefer this approach?